litestar/constants.py

```diff
@@ -1,12 +1,14 @@
 from dataclasses import MISSING
 from inspect import Signature
-from typing import Final
+from typing import Any, Final
 
 from msgspec import UnsetType
 
 from litestar.enums import MediaType
 from litestar.types import Empty
+from litestar.utils.deprecation import warn_deprecation
 
+CONNECTION_STATE_KEY: Final = "_connection_state"
```

Comment:
Maybe we can move these internal constants into a `_constants.py` or mark them as private with `_CONNECTION_STATE_KEY`?

Reply:
We can move it to `utils.scope.state` - I agree, best to keep private things out of here.